src/main/resources/messages/RsBundle.properties

```diff
@@ -90,12 +90,94 @@ refactoring.change.signature.name=Change Signature
+settings.rust.color.const.parameter=Parameters//Const parameter
+settings.rust.color.constant=Variables//Constant
+settings.rust.color.crate=Types//Crate
+settings.rust.color.enum.variant=Types//Enum variant
+settings.rust.color.enum=Types//Enum
+settings.rust.color.field=Variables//Field
+settings.rust.color.format.parameter=Literals//Strings//Format parameter
+settings.rust.color.format.specifier=Literals//Strings//Format specifier inside format parameter
+settings.rust.color.function.call=Functions//Function call
+settings.rust.color.function.declaration=Functions//Function declaration
+settings.rust.color.generated.items=Items generated by macros
+settings.rust.color.invalid.escape.sequence=Literals//Strings//Escape sequence//Invalid
+settings.rust.color.keyword.unsafe=Keywords//Unsafe
+settings.rust.color.keyword=Keywords//Keyword
+settings.rust.color.lifetime=Parameters//Lifetime
+settings.rust.color.macro=Functions//Macro
+settings.rust.color.method.call=Functions//Method call
+settings.rust.color.method.declaration=Functions//Method declaration
+settings.rust.color.module=Types//Module
+settings.rust.color.mutable.binding=Variables//Mutable binding
+settings.rust.color.mutable.parameter=Parameters//Mutable parameter
+settings.rust.color.number=Literals//Number
+settings.rust.color.operation.sign=Braces and Operators//Operation sign
+settings.rust.color.parameter=Parameters//Parameter
+settings.rust.color.primitive=Types//Primitive
+settings.rust.color.question.mark=Braces and Operators//? operator
+settings.rust.color.rustdoc.bold=Rustdoc//Bold text
+settings.rust.color.rustdoc.code=Rustdoc//Code
+settings.rust.color.rustdoc.comment=Rustdoc//Comment
+settings.rust.color.rustdoc.heading=Rustdoc//Heading
+settings.rust.color.rustdoc.italic=Rustdoc//Italic text
+settings.rust.color.rustdoc.link=Rustdoc//Link
+settings.rust.color.scheme.title=Rust
+settings.rust.color.self.parameter=Parameters//Self parameter
+settings.rust.color.string=Literals//Strings//String
+settings.rust.color.struct=Types//Struct
+settings.rust.color.trait=Types//Trait
+settings.rust.color.type.alias=Types//Type alias
+settings.rust.color.type.parameter=Parameters//Type parameter
+settings.rust.color.union=Types//Union
+settings.rust.color.unsafe.code=Unsafe code
+settings.rust.color.valid.escape.sequence=Literals//Strings//Escape sequence//Valid
+settings.rust.color.variables.default=Variables//Default
+
 settings.rust.completion.suggest.out.of.scope.items=Suggest out-of-scope items
 settings.rust.completion.title=Rust
 
+settings.rust.external.linters.additional.arguments.comment=Additional arguments to pass to <b>cargo check</b> or <b>cargo clippy</b> command
+settings.rust.external.linters.additional.arguments.label=Additional arguments:
+settings.rust.external.linters.name=External Linters
+settings.rust.external.linters.on.the.fly.comment=Enable external linter to add code highlighting based on the used linter result. Can be CPU-consuming
```

Comment:
Since it's a note below the checkbox, how about we say this way: "Adds highlighting based on the external linter results. May affect the IDE performance."